examples/subprocess-output/subprocess-output.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+const React = require('react');
+const {render, Text, Box} = require('../../build');
+const childProcess = require('child_process');
+
+const SubprocessOutput = () => {
+	const [output, setOutput] = React.useState('');
+
+	React.useEffect(() => {
+		const subProcess = childProcess.spawn('ping', ['8.8.8.8']).stdout;
+
+		subProcess.on('data', newOutput => {
+			setOutput(newOutput.toString('ascii'));
+		});
```

Comment:
To keep it explicit, because `stdout` is not an instance of the process, so `subProcess` wouldn't be a right name for it.

```suggestion
		const subProcess = childProcess.spawn('ping', ['8.8.8.8']);

		subProcess.stdout.on('data', newOutput => {
			setOutput(newOutput.toString('ascii'));
		});
```